### ISSUE TITLE
Audio streams on PURL but not on exhibit page

### ISSUE THREAD

**Issue by caaster** [sul-dlss/exhibits#2494]
I don't know if this is a sul-embed problem (@andrewjbtw) or an Exhibits problem?

[exhibits-feedback] Latina/o/x @ Stanford exhibit feedback

From: "Hannah Frost" <hfrost@stanford.edu>

Why is the audio not streaming on this page (https://exhibits.stanford.edu/latina-o-x-at-stanford/feature/frances-morales)? It plays back as expected on the PURL: https://purl.stanford.edu/kw623sb1354

**Comment by andrewjbtw** [comment]
I am not having a problem streaming the audio on that exhibits page. I've tried both Firefox and Chrome.

**Comment by caaster** [comment]
Of course, now this is working for me in Chrome. And also in Safari and Firefox. @hannahfrost this appears to have possibly been one of those momentary blips. Could you please try again and comment on this ticket? Thank you.

**Comment by andrewjbtw** [comment]
I'm curious what the viewer looked like when the audio didn't stream - was there a message in the viewer? When we've had problems with the streaming server being overloaded, there's been a message in the viewer saying that the item can't be played.

**Comment by hannahfrost** [comment]
Good question, Andrew. I didn’t take a screenshot.

No message in the viewer. The play button was not actionable. I clicked there and around the player, nothing happened.


—
Hannah Frost | Associate Director for Digital Services
Stanford Libraries<https://library.stanford.edu/> | Digital Library Systems and Services
Lathrop Library 384
E: ***@***.*** | C: (650) 799-1352
Pronouns: she/her

On Jul 25, 2024, at 1:22 PM, Andrew Berger ***@***.***> wrote:



I'm curious what the viewer looked like when the audio didn't stream - was there a message in the viewer? When we've had problems with the streaming server being overloaded, there's been a message in the viewer saying that the item can't be played.
